src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -188,6 +204,34 @@ private void saveWindowState(Stage mainStage) {
         Globals.prefs.putDouble(JabRefPreferences.POS_Y, mainStage.getY());
         Globals.prefs.putDouble(JabRefPreferences.SIZE_X, mainStage.getWidth());
         Globals.prefs.putDouble(JabRefPreferences.SIZE_Y, mainStage.getHeight());
+        printWindowState(mainStage);
+    }
+
+    /**
+     * outprints the Data from the Screen
+     * (only in debug mode)
+     * @param mainStage
+     */
+    private void printWindowState(Stage mainStage) {
+        StringBuilder bob = new StringBuilder();
+        bob.append("SCREEN DATA:");
+        bob.append("JabRefPreferences.WINDOW_MAXIMISED: " + mainStage.isMaximized() + "\n");
```

Comment:
Shouldn't `JabRefPreferences` be replaced by `MainStage` ? (As this does not print the values stored in the preferences)

Reply:
yes it should, I think I had it wrong in because of earlier tries, were I also printed the preferences 😅 it'll be changed at the next commit